test/RunTests

```diff
@@ -2485,4 +2485,14 @@ train-sets/ref/slates_simple_w_interactions.stderr
 {VW} -d train-sets/cb_multi_cost.dat --cb 4 -k -c --passes 2 --holdout_off --cb_force_legacy
     train-sets/ref/cb_multi_cost_multipass_legacy.stderr
 
```

Comment:
Windows CI doesn't run json/dsjson tests, you can make the windows ci skip them [here](https://github.com/VowpalWabbit/vowpal_wabbit/blob/master/cs/unittest/RunTests.tt#L24)

Reply:
Thanks!